app/src/main/java/com/rage/clamber/Activities/HomePage.java

```diff
@@ -0,0 +1,24 @@
+package com.rage.clamber.Activities;
+
+import android.os.Bundle;
+import android.support.v4.app.FragmentTransaction;
+import android.support.v7.app.AppCompatActivity;
+
+import com.rage.clamber.Fragments.ActionBarFragment;
+import com.rage.clamber.R;
+
+public class HomePage extends AppCompatActivity {
```

Comment:
It's probably a good idea to add a javadoc comment on the class itself as well, describing overall what it's for and what it's expected to do at a high level.